src/LibraryManager.Vsix/Shared/VsHelpers.cs

```diff
@@ -545,5 +545,17 @@ private static void DeleteEmptyFolders(HashSet<ProjectItem> folders)
                 }
             }
         }
+
+        public static Guid GetProjectGuid(Project project)
+        {
+            string uniqueName = project.UniqueName;
```

Comment:
Should we do some sort of return result checking in any of these? I'd assume there are HRs returned and object could be checked for null.